### DISCUSSION TITLE
Is there any easy way to just get the generated tokens instead of the full output?

### DISCUSSION THREAD

**Discussion by arnavgarg1** [bentoml/OpenLLM#204] [General]
Same as the discussion title - I am wondering if I can just get the generated tokens in the response as opposed to the input tokens + generated tokens.

I couldn't see anything around this here: https://github.com/bentoml/OpenLLM/blob/main/src/openllm/client/runtimes/base.py. Probably just needs a new `return_response` strategy?

**Comment by aarnphm** [comment]
This is working in progress, in the same line with SSE
